### ISSUE TITLE
Disables playing of video completely, sometimes.

### ISSUE THREAD

**Issue by fairypants** [Eloston/disable-html5-autoplay#161]
I find this plug-in works really well. But it seems to mess up the playing function of videos on some sites. Say a video doesn't start automatically, which is what I want usually. But then I hit play and it doesn't start playing until I hit the little timeline bar, which only works about the half the time; the other half, the video just doesn't play at all unless I choose "disable nothing" in the plugin for the page, then refresh the page.

**Comment by Eloston** [comment]
Yeah it's a hit-and-miss with complicated media players. Hopefully future iterations of Chrome/Chromium or the 0.7 rewrite will fix this.
